modules/improvedigitalBidAdapter.js

```diff
@@ -177,11 +189,11 @@ export const spec = {
           cpm: bidObject.price,
           creativeId: bidObject.crid,
           currency: serverResponse.body.cur.toUpperCase() || 'USD',
-          dealId: (typeof idExt.buying_type === 'string' && idExt.buying_type !== 'rtb') ? idExt.line_item_id : undefined,
+          dealId: (idExt && typeof idExt.buying_type === 'string' && idExt.buying_type !== 'rtb') ? idExt.line_item_id : undefined,
           meta: {
             advertiserDomains: bidObject.adomain ? bidObject.adomain : []
           },
-          netRevenue: idExt.is_net || false,
+          netRevenue: idExt && idExt.is_net || false,
```

Comment:
code won't build because of this line

Reply:
<img width="913" alt="image" src="https://user-images.githubusercontent.com/93644923/162893671-11651f64-5e99-46bc-8bb0-a6cad48cb538.png">